Kinesis log collection images (#8354)
DataDog/documentation: content/en/logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination.md

5. Select the `Datadog` destination and select the `Datadog US` or `Datadog EU` region, depending on the Datadog Region of your account.
  {{< img src="logs/guide/choose-destination.png" alt="Choose your destination" style="width:100%;">}}
6. Paste your `APIKEY` into the `AccessKey` box. (You can get your API key from [your Datadog API settings page][3]).

> 7. (Optional) Add custom `parameters`—these get added as custom tags on your logs.

Reviewer: ```suggestion
7. (Optional) Add custom `parameters`, which are added as custom tags to your logs.
```

{{< img src="logs/guide/kinesis_logs_datadog_destination.png" alt="Datadog destination configuration" style="width:100%;">}}
8. Choose to backup failed events to an S3 bucket.

> 9. Configure the Delivery stream parameters. The two important parameters:

Reviewer: ```suggestion
9. Configure the delivery stream parameters. The two important parameters are:
```
